### PR TITLE
TestingSupport::ControllerRequests doc current practice and effects

### PR DESCRIPTION
**Pull request by jrochkind**

### CODE REVIEW

File: core/lib/spree/testing_support/controller_requests.rb [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
   module TestingSupport
-    # Use this module to easily test Spree actions within Spree components or
-    # inside your application to test routes for the mounted Spree engine.
+    # A module providing convenience methods to test Solidus controllers
+    # in Rails controller/functional tests. Possibly from inside an
+    # application with a mounted Solidus engine.
     #
-    # Inside your spec_helper.rb, include this module inside the
-    # RSpec.configure block by doing this:
+    # *There is generaly no need* to use this module.* Instead, in
```
Comment (jhawthorn) [inline core/lib/spree/testing_support/controller_requests.rb]: Is the second `*` here a typo?

Comment (jrochkind) [inline core/lib/spree/testing_support/controller_requests.rb]: yes

**Comment by jhawthorn** [comment]
:+1:

**Comment by mamhoff** [comment]
:+1: Nice work!

**Pull request merged**